backend/.ocamlformat-enable

```diff
@@ -1,2 +1,8 @@
 cmm_helpers.ml
 cmm_helpers.mli
+cfg/*.ml
+cfg/*.mli
+asm_targets/*.ml
+asm_targets/*.mli
+debug/dwarf/*.ml
+debug/dwarf/*.mli
```

Comment:
Does this need to list the subdirectories of `dwarf/` too?

Reply:
actually, could we change this to cover `debug/*.ml{,i}` even if there are no files matching that at present?

Reply:
Do you mean to cover `debug/dwarf/*.ml{,i}` ?

Reply:
No, I mean directly under `debug/`, since we will add files in there in due course.